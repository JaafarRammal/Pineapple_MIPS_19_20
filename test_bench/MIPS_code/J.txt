J 0X4
ADDI R2, R2, 0X2
ADDI R2, R2, 0X3
ADDI R2, R2, 0X9
JR R0