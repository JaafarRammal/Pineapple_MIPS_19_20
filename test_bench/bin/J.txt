00001000000000000000000000000010
00100000010000100000000000000010
00100000010000100000000000000011
00100000010000100000000000001001
00000000000000000000000000001000