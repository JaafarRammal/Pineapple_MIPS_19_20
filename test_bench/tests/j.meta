#J
#4
#MaelleGuerre
#basic jump test
#
#

addi $1, $1, 8
j target
addi $3, $3, 8192
addi $2, $2, 2
target:
addi $2, $2, 2
addi $2, $2, 2
jr $0