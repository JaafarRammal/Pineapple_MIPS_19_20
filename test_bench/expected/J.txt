9
2 or 5